sys/net/gnrc/link_layer/netdev2/gnrc_netdev2_eth.c

```diff
@@ -114,12 +114,11 @@ static inline void _addr_set_broadcast(uint8_t *dst)
 static inline void _addr_set_multicast(uint8_t *dst, gnrc_pktsnip_t *payload)
 {
     switch (payload->type) {
-#ifdef MODULE_IPV6
+#ifdef MODULE_GNRC_IPV6
         case GNRC_NETTYPE_IPV6:
             dst[0] = 0x33;
             dst[1] = 0x33;
             memcpy(dst + 2, ((uint8_t *)payload->data) + _IPV6_DST_OFFSET, 4);
-            /* TODO change to proper types when gnrc_ipv6_hdr_t got merged */
```

Comment:
Maybe add a link to https://tools.ietf.org/html/rfc2464#section-7 in the comments so I don't have to google it next time ^^